extensions/vault/runtime/src/main/java/io/quarkus/vault/VaultRecorder.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.vault;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.Arc;
+import io.quarkus.runtime.annotations.Recorder;
+import io.quarkus.vault.config.VaultServerRuntimeConfig;
+
+@Recorder
+public class VaultRecorder {
```

Comment:
This is not API, it should be in the `io.quarkus.vault.runtime` subpackage. Only APIs should be at the root package.

Same remark for everything that is not API.

Reply:
everything has been moved to a runtime package (which was named internal originally).
I only kept the VaultKvService interface and the base exception in the base package.